apollo-client/pom.xml

```diff
@@ -97,6 +97,17 @@
 			<artifactId>mockserver-netty</artifactId>
 			<scope>test</scope>
 		</dependency>
+    <dependency>
+      <groupId>com.github.noconnor</groupId>
+      <artifactId>junitperf</artifactId>
+      <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>ch.qos.logback</groupId>
+          <artifactId>logback-classic</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
```

Comment:
The indent format appears incorrect.

Reply:
Sorry, it looks like there was a problem synchronizing the remote code with my local code. This has been fixed.